### DISCUSSION TITLE
Welcome to Validation Discussions!

### DISCUSSION THREAD

**Discussion by alganet** [Respect/Validation#1366] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dcorrea777** [comment]
The first time I started contributing to Respect\Validation was at an event that took place in São Paulo, I think the event name was TestFestValidation, something like that.

On the day of the event, I remember there was @henriquemoody , @alganet  and @williamespindola , and other people from the community, so Henrique made a basic introduction about Respect\Validation and how we could contribute to the project.

For me that was really amazing, because I've never contributed to an open source project, and in a few minutes I had submitted a lot of PR =)

After a few years without contributing to the project, I found a task in which Henrique asked for help from the community for the release of version 2.0, that's where I made most of my contributions.

I have a lot of affection for this project because it's actually where I started contributing open source. ❤️

**Comment by reformed** [comment]
Hello,

There's a typo on the [HexRgbColor page](https://respect-validation.readthedocs.io/en/latest/rules/HexRgbColor/):

> Validates weather the input is a hex RGB color or not.

Should be

> Validates whether the input is a hex RGB color or not.

**Reply by henriquemoody** [reply]
Thank for that! I just fixed it!

**Comment by TonyGravagno** [comment]
Thanks for creating this area for discussion!

My primary focus these days is on applications that use WordPress as a platform/CMS. I have spent many months studying ReactJS in-depth so that I can work on the front-end of these apps. React has JS validation tools like Zod and Yup. At some point soon I need to swing back to the PHP back-end, where some data comes from common WordPress UI components but data for my apps will come in via REST and that data Must be cleansed and validated in terms of type and application context. That's where I will (re)begin my experience with Respect/Validation.

I look forward to this adventure. Thanks!
